source/_docs/terminus/plugins/create.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Terminus Create Plugins
+terminuspage: true
+terminuscreate: true
+terminustoc: true
+type: terminuspage
+layout: terminuspage
+permalink: docs/terminus/plugins/:basename/
+---
+Add new commands to Terminus by creating a plugin. The following explains how to create our [example plugin](https://github.com/pantheon-systems/terminus-plugin-example) to demonstrate how to add new commands.
+
+## Create the Example Plugin
+This example provides the structural requirements for plugins to be recognized and loaded by Terminus.
+### 1. Create Plugin Directory
+A plugin is a specific set of files bundled together in a directory. The first step in creating a new plugin is to create that directory. It does not matter what you name your plugin’s directory but it makes sense to give it the same name as your plugin. Plugins must live within `$HOME/.terminus/plugins`. You may need to create the `$HOME/.terminus/plugins` directory if it does not already exist.
+
+Create a directory for your new plugin (e.g. `hello-world`):
+<div class="copy-snippet">
+<button class="btn btn-default btn-clippy" data-clipboard-target="#create-plugin-dir"><img class="clippy" src="/source/docs/assets/images/clippy.svg" width="17" alt="Copy to clipboard"></button>
+<figure><pre id="create-plugin-dir"><code class="bash" data-lang="bash">mkdir $HOME/.terminus/plugins/hello-world</code></pre></figure>
+</div>
+
+### 2. Create composer.json
+In order for Terminus to recognize the directory as a plugin, the directory must contain a valid `composer.json` file. Download the following file and move it to your plugin's root directory (e.g. `$HOME/.terminus/plugins/hello-world`):
+<div class="script-file-header">
+composer.json
+<a id="downloadLink"><button class="btn btn-default btn-download"><i class="fa fa-download" aria-hidden="true"></i>   Download File
+</button></a>
+</div>
+<pre><code id="composer">{
+    "name": "terminus-hello-world",
```

Comment:
Composer names should be "org/name", so this could something like:
```
"name": "my-github-org/terminus-hello-world",
```
`my-github-org` should just be your github user if not using an org. Also, folks won't necessarily be using github.  Just `my-username`?